addon/-private/ember-internals.js

```diff
@@ -1,16 +1,21 @@
 import Ember from 'ember';
 
 let __EMBER_METAL__;
-if (Ember.__loader.registry['@ember/-internals/metal']) {
-  __EMBER_METAL__ = Ember.__loader.require('@ember/-internals/metal');
+let emberMetalPaths = [
+  '@ember/-internals/metal', // ember-source from 3.10
+  '@ember/-internals/metal/index' // ember-source from 3.13
+];
+let metalPath = emberMetalPaths.find(path => Ember.__loader.registry[path]);
+if (metalPath) {
+  __EMBER_METAL__ = Ember.__loader.require(metalPath);
 }
 
 export function getDependentKeys(descriptorOrDecorator) {
   if (__EMBER_METAL__ && __EMBER_METAL__.descriptorForDecorator) {
     let descriptor = __EMBER_METAL__.descriptorForDecorator(
       descriptorOrDecorator
     );
-    return descriptor._dependentKeys;
+    return descriptor._dependentKeys || [descriptor.altKey];
```

Comment:
What scenario is `altKey` used for?

Reply:
I'm not sure TBH. There are failing tests without it